### ISSUE TITLE
Installation check fails on notebook

### ISSUE THREAD

**Issue by stargaser** [NASA-NAVO/navo-workshop#117]
Reported by @alaity47 

`python check_env.py` fails to find module "notebook". 

```
(navo-env) garibaldi:navo-workshop laity$ python check_env.py 
Found jupyter 
Error: Failed import: No module named 'notebook'
Found numpy 1.24.3
Found matplotlib 3.7.1
Found jupyterlab 4.0.0
Found astropy 5.2.2
Found pyvo 1.4.1
Found astroquery 0.4.6

There are errors that you must resolve before running the tutorials.
```

Also, we don't check for the existence of the "scipy" module even though it is installed in the notebook.

Notes:
* Newest Intel miniconda threw an "incompatible architecture" error. Second-newest miniconda worked.
* Step 5 of the [installation instructions](https://nasa-navo.github.io/navo-workshop/00_SETUP.html) took a long time to run some of its pieces, might want to to note it's expected to spin for a while.

**Comment by bsipocz** [comment]
Maybe time to switch to mamba? That may resolve step5 being too slow.

**Comment by stargaser** [comment]
Indeed, I switched the astropy-workshop installation instructions to use mamba. Could copy those instructions.
